server/controller/user-contoller.js

```diff
@@ -21,21 +22,22 @@ const signUp = async(req,res,next) =>{
    const { name , email , password } = req.body;
```

Comment:
Consider disabling autocomplete for the password field to prevent the browser from storing or suggesting passwords, which could enhance security.

```suggestion
const { name , email , password } = req.body; // autocomplete="off"
```